apps/docs/content/3-components/2-library/38-tag.mdx

```diff
@@ -2,13 +2,70 @@
 title: Tag
 description: Tag HTML Component
 draft: false
-status: in-development
+status: stable
```

Comment:
@pregno 

Let me know if you think I can update it to stable?

Reply:
leave it `in-development` at least until we release the Card